modules/oxide/src/less/theme/components/button/button.less

```diff
@@ -2,6 +2,72 @@
 // Button
 //
 
+.tox-button when (@custom-properties-enabled = true) {
+  //Base variables
+  --tox-private-button-background-color: var(--tox-private-color-tint);
+  --tox-private-button-text-color: var(--tox-private-color-white);
+  
+  --tox-private-button-border-width: 1px;
+  --tox-private-button-border-style: solid;
+  --tox-private-button-border-color: var(--tox-private-button-background-color);
+  
+  --tox-private-button-padding-x: var(--tox-private-pad-md);
+  --tox-private-button-padding-y: var(--tox-private-pad-xs);
+  
+  --tox-private-button-box-shadow: none;
+
+  --tox-private-button-font-size: var(--tox-private-font-size-sm);
+  --tox-private-button-font-style: normal;
+  --tox-private-button-font-weight: var(--tox-private-font-weight-bold);
+
+
+  // Button state variables - to be decided if we want them. 
+  // If we do, they should inherit from button base variables (as they do now). While skinning the change of the base variable will result with others being calculated/inherited from it. In this case it is also possible to skin each state separately. 
+  // If we don't, they should be removed. Different states of the button should use base button variables and calculate from them if neccessary, but not create new variables. In this case it is NOT possible to skin each state separately. 
```

Comment:
I think these two comments are too long. Maybe they need to be split up across more lines